cohortextractor/emis_backend.py

```diff
@@ -972,7 +972,11 @@ def patients_with_these_codes_on_death_certificate(
                 hashed_organisation,
                 {column_definition} AS {column_name}
             FROM ons_view
-            WHERE ({code_conditions}) AND {date_condition}
+            WHERE ({code_conditions})
+                AND {date_condition}
+                AND date_parse(upload_date, '%d/%m/%Y') = (
```

Comment:
Add a comment here? 

Also if you've not already checked this, verify this is what TPP are already doing for us, and mention this behaviour in a short sentence [in the SGSS documentation](https://docs.opensafely.org/en/latest/dataset-sgsscovid/)?

Reply:
> verify this is what TPP are already doing for us

Done.